app/src/main/java/com/example/cocktails/ScrollingActivity.kt

```diff
@@ -5,24 +5,20 @@ import android.app.SearchManager
 import android.content.Context
 import android.content.Intent
 import android.graphics.Color
-import android.net.Uri
 import android.os.Bundle
 import android.os.Parcelable
-import com.google.firebase.auth.FirebaseUser
-import android.widget.Toast
 import android.view.Menu
 import android.view.MenuItem
 import android.view.View
-import android.widget.AutoCompleteTextView
-import android.widget.EditText
```

Comment:
@enavhasson why? I'm using those classes, see 'initSearchView' method for example.
I think you need to revert this one